setup.py

```diff
@@ -180,12 +180,12 @@ def run(self):
             if is_manylinux:
                 file = glob(path.join(self.dist_dir, '*linux*.whl'))[0]
                 logger.info('repairing %s for manylinux1', file)
-                try:
-                    subprocess.run(['auditwheel', 'repair', '-w', self.dist_dir, file],
-                                   check=True, stdout=subprocess.PIPE)
-                finally:
-                    logger.info('removing %s', file)
-                    remove(file)
+                # try:
```

Comment:
Talked with @ajindal1: we will delay this change until the 1.9 release is finished, because this change will affect all the python packaging pipelines.